src/test/java/uk/gov/hmcts/reform/translate/service/DictionaryServiceTest.java

```diff
@@ -101,16 +118,98 @@ void shouldThrowExceptionWhenDictionaryContainsDuplicateEnglishPhrases() {
     }
 
     private DictionaryEntity createDictionaryEntity(String phrase, String translationPhrase) {
-        return DictionaryEntity.builder()
-            .englishPhrase(phrase)
-            .translationPhrase(translationPhrase)
-            .build();
+        final var dictionaryEntity = new DictionaryEntity();
+        dictionaryEntity.setEnglishPhrase(phrase);
+        dictionaryEntity.setTranslationPhrase(translationPhrase);
+        return dictionaryEntity;
     }
 
     @Test
     void shouldReturnEmptyDictionaryContents() {
         given(dictionaryRepository.findAll()).willReturn(repositoryResults);
         assertTrue(dictionaryService.getDictionaryContents().isEmpty());
     }
+
+
+    @Test
```

Comment:
Admittedly I didn't do this for the GET endpoint (WLTS-5), but now that this class is going to get quite large would be good to put the `PUT` endpoint tests in a junit `Nested` class ?

Reply:
NICE ONE DONE